patricktunez123/Andela_Teamwork
# #168760540 Add css to fix colors issues for ui

#### What does this PR do?
Update UI pages
#### Description of Task to be completed?
Have the ui pages having the same color and fonts
#### How should this be manually tested?
In URL type https://patricktunez123.github.io/Andela_Teamwork/UI/ 
click login button you should see colors are fixed
#### What are the relevant pivotal tracker stories?
#168760540
